composer.json

```diff
@@ -72,7 +72,7 @@
         "drupal/recaptcha": "^3",
         "drupal/redirect": "^1",
         "drupal/scheduler": "^1",
-        "drupal/search_api": "1.17",
+        "drupal/search_api": "1.17 || ^1.19",
```

Comment:
1.18 is not allowed?

Reply:
@duozersk because we have no resources to test all versions - and we can only test latest one today - no, it's not allowed
But if you need to have it
`composer require "drupal/search_api":"1.18 as 117"`

Reply:
Then why do we need this special care about 1.17? I would be fine with just "^1.19". And if anyone wants to have anything older than that (1.18 or 1.17) - then use aliases...

Reply:
@duozersk because we keep tracking the previous version in these ORs
https://github.com/ymcatwincities/openy/wiki/Composer-version-constraints-for-Open-Y 
In order to support developers with additional Stable state if latest one failed for their case

Reply:
@podarok In this case maybe better to hardcode the `1.19` version too? Future versions can be updated without testing

Reply:
ok, good-good

Reply:
@hamrant a while ago we decided to not lock versions from upgrades. This eliminates a need to fire a new OpenY release every time there is an update of the respective module or critical security update of it.
For old versions, we are safe to go with specific version points, for the latest versions we are opening options.
We sometimes were locking some dependencies, especially when they contained breaking changes and lack of upgrade path support ( drupal/plugin, search_api 1.x , etc ), but recently we found all good in a Drupal/OpenY world of dependencies

Also, we suggest `--prefer-stable` or even `--prefer-lowest` while doing composer runs if there is a requirement to do really conservative.

Reply:
Ok, thanks, sound good for me